lib/modules/manager/npm/post-update/utils.ts

```diff
@@ -32,6 +32,12 @@ export function getPackageManagerVersion(
   name: string,
   pkg: PackageJsonSchema,
 ): string | null {
+  if (pkg.volta?.[name]) {
```

Comment:
btw: do we pass the workspace root package here or the local? 🤔

Reply:
Local

Reply:
so we probably need to check the parent package.json if we're in a workspace

Reply:
That's why we sometimes see wrong tool version usage 🤦‍♂️

Reply:
Ideally Renovate should use the workspace member tools for the workspace member itself, but I agree relying on the workspace parent tools is miles better than the current behavior.